apps/gauzy/src/app/pages/settings/oauth-clients/oauth-client-secret-dialog/oauth-client-secret-dialog.component.html

```
@@ -0,0 +1,42 @@
+					<nb-icon icon="copy-outline"></nb-icon>
+				</button>
+			</div>
+		</div>
+
+		<nb-toggle [(checked)]="acknowledged">
+			{{ 'OAUTH_CLIENTS.SECRET_DIALOG.ACKNOWLEDGE' | translate }}
+		</nb-toggle>
+	</nb-card-body>
+
+	<nb-card-footer class="actions">
+		<button nbButton status="primary" [disabled]="!acknowledged" (click)="close()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)